search/search_query.cpp

```diff
@@ -125,16 +125,25 @@ class IndexedValue : public search::IndexedValueBase<Query::kQueuesCount>
   shared_ptr<impl::PreResult2> m_val;
 
   double m_rank;
+  double m_distanceToPivot;
 
 public:
   explicit IndexedValue(unique_ptr<impl::PreResult2> v)
-    : m_val(move(v)), m_rank(m_val ? m_val->GetRankingInfo().GetLinearModelRank() : 0)
+    : m_val(move(v)), m_rank(0), m_distanceToPivot(numeric_limits<double>::max())
   {
+    if (!m_val)
+      return;
+
+    auto info = m_val->GetRankingInfo();
```

Comment:
This should be replaced by GetLinearModeRank. No hurry, I'm going to fix this tomorrow.